analytics/lib/counts.py

```diff
@@ -343,16 +347,26 @@ def do_increment_logging_stat(
     else:
         raise AssertionError("Unsupported CountStat frequency")
 
-    row, created = table._default_manager.get_or_create(
-        property=stat.property,
-        subgroup=subgroup,
-        end_time=end_time,
-        defaults={"value": increment},
-        **id_args,
+    id_args_keys = SQL(", ").join(map(Identifier, id_args.keys()))
+    values = SQL(", ").join(map(Literal, id_args.values()))
+    sql_query = SQL(
+        """
+        INSERT INTO {table_name}(property, subgroup, end_time, value, {id_args_keys})
+        VALUES (%s, %s, %s, %s, {id_args_values})
+        ON CONFLICT (property, subgroup, end_time, {id_args_keys_update})
+        DO UPDATE SET
+        value = {table_name}.value + %s
```

Comment:
You can say this as:
```suggestion
        value = {table_name}.value + excluded.value
```
...which removes the need to specify the increment more than once.  See https://www.postgresql.org/docs/current/sql-insert.html#SQL-ON-CONFLICT and search for `excluded`